### ISSUE TITLE
data is not clearing from cache

### ISSUE THREAD

**Issue by ushaNS** [iainconnor/ObjectCache#14]
i have created a object cache for 2 hours (ex: 8am to 10am) , when 2 hours get completed data will disappear in app as expected, but problem is their with following scenario

after 2 hours (i.e 8am to 10am) gets completed data will be cleared from app, but when i reset the time to anytime between 2 hours (i.e 8am to 10am) again data will get displayed on android activity screen.

that means data is not clearing from memory after specified time exceeds.

**Comment by qteen1** [comment]
@ushaNS yeah I'm having the same problem as well. I uploaded an image as a byte array to the disk and I set the timer for 1 minute. After 1 minute, when i replug my android device back into my laptop and look at the directory where I uploaded my image it is still there. It seems the timer does not work :/. Were you able to get it working?
